packages/botonic-react/src/webchat/context/index.tsx

```diff
@@ -54,6 +54,9 @@ export const WebchatContext = createContext<WebchatContextProps>({
   togglePersistentMenu: () => {
     return
   },
+  updateCustomJsonMessage: () => {
```

Comment:
rename to `updateCustomMessageJSON` so its clear that it references `CustomMessage`'s

Reply:
applied